react/week2/exersice-react/src/components-todo/TodoListItem.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function TodoListItem(props) {
+
+        return (
+            <li>
+                <p >{props.id}. {props.description}, {props.date}</p>
+                <button className='delete-button'>Delete</button>
+            </li>
+        );
+    }
```

Comment:
You can destructure the props object like this so it would be clear, neat and you don't always have to write props at first by this way. As we have passed the removeTodo function, we are using it here to delete todo. 
```suggestion
function TodoListItem({ todo, removeTodo }) {
  return (
    <li id={todo.id} key={todo.id}>
      <p>
        {todo.id}. {todo.description}, {todo.date}
      </p>
      <button className="delete-button" onClick={() => removeTodo(todo.id)}>
        Delete
      </button>
    </li>
  );
}
```